### ISSUE TITLE
Refresh page impagination is not corrent

### ISSUE THREAD

**Issue by franklupo** [fgilde/MudBlazor.Extensions#8]
HI,
If you refresh page F5 the layout is not correct.

![image](https://user-images.githubusercontent.com/6276309/201352140-deb12ef0-5d82-41a8-a367-d264dfe8e3c1.png)

after refresh 

![image](https://user-images.githubusercontent.com/6276309/201352354-7056868e-eda6-4f29-b7c7-7c832bf23a0d.png)

Best reagrds

**Comment by fgilde** [comment]
Reloading in general is not a problem and works as expected. Maybe you have some error in console. You need to check or some css selectors are affected or something like this. On my projects its working fine also on https://mudex.azurewebsites.net/object-edit

Could you please check your console, or send me a sample solution where this problem exists?

**Comment by franklupo** [comment]
Hi,
I have no errors.
![image](https://user-images.githubusercontent.com/6276309/201610602-f039dc0c-7c46-4003-9b68-5efadbb55e42.png)

Using package 6.1.0
```
<PackageReference Include="MudBlazor" Version="6.1.0" />
```

i added initialization in app.razor.cs but this is in external library which is loaded.

Best regards

**Comment by franklupo** [comment]
Hi,
my project use a razor library.
The MainLayout and initialization it is in a razor library not in main project.
This could be the problem?

best regards

**Comment by fgilde** [comment]
Not sure, can you create a small solution where the problem occurs, and send it to me?

**Comment by fgilde** [comment]
Thanks. I will check it asap

**Comment by fgilde** [comment]
Ok, there is a problem on my side if application is ServerPrerendered. (sorry didn't check before I always use WebAssembly) 
However I#ve created a new Version with a workaround for you.  Use Version 1.7.11
`<PackageReference Include="MudBlazor.Extensions" Version="1.7.11" />`

And pass then `true` to the `InitializeMudBlazorExtensionsAsync` Method.

```c#

    protected override async Task OnAfterRenderAsync(bool firstRender)
    {
        if (firstRender)
            await JsRuntime.InitializeMudBlazorExtensionsAsync(true);
        await base.OnAfterRenderAsync(firstRender);
    }
```

I will change this later on with a better solution, but hopefully that should help first


Working now.

[MudBlazorTemplates3.zip](https://github.com/fgilde/MudBlazor.Extensions/files/10123683/MudBlazorTemplates3.zip)

**Comment by franklupo** [comment]
Good work.

Best regards

**Comment by franklupo** [comment]
everything works perfect

**Comment by fgilde** [comment]
closed with 1.7.11
